python/paddle/tensor/logic.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import paddle
```

Comment:
I'm not sure whether here will cause "import each other" problem. I saw they just used "from xxx import xxx". Let's see whether CI tests pass. If you can, please change to "from xxx import xxx" to match styles.

Reply:
Modified.